modyn/selector/internal/selector_strategies/presampling_strategies/random_presampling_strategy.py

```diff
@@ -0,0 +1,51 @@
+from typing import Optional
+
+from modyn.metadata_database.models import SelectorStateMetadata
+from modyn.selector.internal.selector_strategies.presampling_strategies.abstract_presampling_strategy import (
+    AbstractPresamplingStrategy,
+)
+from sqlalchemy import Select, asc, func, select
+
+
+class RandomPresamplingStrategy(AbstractPresamplingStrategy):
```

Comment:
more like `UniformPresampling`, right?

Reply:
Yep, all the presampling is uniform since there's no way to so something more advanced directly on the DB. I think that Uniform is implicit, but I can change if you prefer